cmake/Helper.cmake

```diff
@@ -6,7 +6,7 @@ if(CMAKE_BUILD_TYPE STREQUAL Release OR CMAKE_BUILD_TYPE STREQUAL RelWithDebInfo
   if(NOT WASMEDGE_FORCE_DISABLE_LTO)
     set(WASMEDGE_INTERPROCEDURAL_OPTIMIZATION ON)
   endif()
-  if (CMAKE_GENERATOR STREQUAL Ninja)
+  if (CMAKE_GENERATOR STREQUAL Ninja AND NOT ANDROID)
```

Comment:
You don't need to add extra check, because you are not using `ninja` building system.